src/api/routes/subscriptions.js

```diff
@@ -379,9 +379,11 @@ subscriptions.addSubscription = function (app, res, applications, loggedInUserId
                     }
 
 
+                    const actionType = subsCreateInfo.ModifyPlan ? webhooks.ACTION_MODIFY : webhooks.ACTION_ADD;
```

Comment:
what happens if there is no modifyPlan field in subscreateInfo obj ? do you see any exception ?